components/Tabs/Tabs.js

```diff
@@ -1,62 +1,93 @@
+class Tabs {
+    constructor(element) {
+        this.element = element;
+        this.tabObject = new TabLink(this.element);
+        element.addEventListener('wheel', () => this.deselect());
+    }
+
+    // Methods
+    deselect() {
+        if (this.element.classList.contains('tabs-link-selected' || this.element.classList.contains('tabs-item-selected'))) {
+            const links = document.getElementsByClassName('tabs-link');
+            const items = document.querySelectorAll('.tabs-item');
+            Array.from(links).forEach(element => element.classList.remove('tabs-link-selected'));
+            Array.from(items).forEach(element => element.classList.remove('tabs-item-selected'));
+
+        }
+    }
+}
+
+
 
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
-    
+
+    this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
+    //console.log(this.data);
+  
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+    this.itemElement = document.querySelector(`.tabs-item[data-tab='${this.data.dataset.tab}'`);
+    //console.log(this.itemElement);
+     
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
+    this.itemObject = new TabItem(this.itemElement);
+    //console.log(this.itemElement);
     
     // Add a click event listener on this instance, calling the select method on click
-
+    element.addEventListener('click', () => this.select());
   };
 
   select() {
     // Get all of the elements with the tabs-link class
-    // const links;
+    const links = document.getElementsByClassName('tabs-link');
 
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
+    Array.from(links).forEach(element => element.classList.remove('tabs-link-selected'));
 
     // Add a class named "tabs-link-selected" to this link
-    // this.element;
+    this.element.classList.add('tabs-link-selected');
     
     // Call the select method on the item associated with this link
-
+    this.itemObject.select();
   }
 }
 
+
+
 class TabItem {
   constructor(element) {
     // Assign this.element to the passed in element
-    // this.element;
+    this.element = element;
   }
 
   select() {
     // Select all ".tabs-item" elements from the DOM
-    // const items;
+    const items = document.querySelectorAll('.tabs-item');
+    console.log(items);
 
     // Remove the class "tabs-item-selected" from each element
+    Array.from(items).forEach(element => element.classList.remove('tabs-item-selected'));
```

Comment:
Though it still functions, `Array.from()` is not needed here because `querySelectorAll` returns a NodeList.